torchchat/generate.py

```diff
@@ -39,11 +43,16 @@
     BuilderArgs,
     TokenizerArgs,
 )
-from torchchat.distributed.generate import DistributedGenerator
+from torchchat.distributed.utils import (
+    Color as color,
+    setup_env,
```

Comment:
nit: `setup_env` is pretty generic, let's alias it so it's obviously for a distributed context